composables/discover.ts

```diff
@@ -10,7 +10,7 @@ export type IDiscoverableGroup = {
   items: Exclude<IAllDocumentModels, SectionHeaderModel>[];
 };
 
-export function useDiscover(requestParameters: DiscoverGetRequest) {
+export function useDiscoverOld(requestParameters: DiscoverGetRequest) {
```

Comment:
This does not seem to be used anywhere... Can it be removed?

Reply:
removed